wiv.js

```diff
@@ -95,31 +109,31 @@ function wiv() {
     let oY = y;
 
     //draw top
-    for (x = height * 3; x <= canvas.width - (height * 3); x += 1) {
+    for (x = height * 3; x <= canvas.width - (height * 3); x += increment) {
       y = height - Math.sin(((x - frame) * tightness) * Math.PI / 180) * height + thickness;
       ctx.lineTo(x, y);
     }
 
     //draw right
-    for (y = y; y <= canvas.height - (height * 3); y += 1) {
+    for (y = y; y <= canvas.height - (height * 3); y += increment) {
       x = (canvas.width - height * 3) + height - Math.cos(((y - frame) * tightness) * Math.PI / 180) * height + thickness;
       ctx.lineTo(x, y);
     }
 
     //draw bottom
-    for (x = x; x >= (height * 3); x -= 1) {
+    for (x = x; x >= (height * 3); x -= increment) {
       y = (canvas.height - height * 3) + height - Math.sin(((x - frame) * tightness) * Math.PI / 180) * height + thickness;
       ctx.lineTo(x, y);
     }
 
     //draw left
-    for (y = y; y >= (height * 2) + thickness; y -= 1) {
+    for (y = y; y >= (height * 2) + thickness; y -= increment) {
       x = height - Math.cos(((y - frame) * tightness) * Math.PI / 180) * height + thickness;
       ctx.lineTo(x, y);
     }
 
     //draw top
-    for (x = x; x <= canvas.width - (height * 3); x += 1) {
+    for (x = x; x <= (height * 3) + increment; x += increment) {
```

Comment:
@jjkaufman: another small change here. This second pass at the top line only needs to run for the very beginning before it starts overlapping with the first pass at the top line.

Reply:
@bendemeyer perfect thank you 

Hopefully we can eliminate the need for this with proper intersections